docs/content/contribute/review-pr.md

```diff
@@ -18,19 +18,21 @@ This page provides guidelines for reviewing a Magic Modules pull request (PR).
       * features only available in beta are not included in the GA google provider.
       * features added to the GA provider are also included in the beta provider -- beta should be a strict superset of GA.
    1. no [breaking changes]({{< ref "/develop/make-a-breaking-change" >}}) are introduced without a valid justification. Add the `override-breaking-change` label if there is a valid justification.
-   1. verify the change actually resolves the linked issues, if any.
+      * remember to check for changes in default behaviour like changing the flags on delete! 
+   1. verify the change **fully** resolves the linked issues, if any. If it does not, change the "Fixes" message to "Part of".
```

Comment:
Does breaking-change-detector catch changes like these currently?

Reply:
No- it operates on the schema of the resource, and a change like this would happen inside of the resource implementation. The detector can only catch surface-level stuff, unfortunately.

Reply:
(same idea as how API tooling can catch removed fields, but can't catch things like fields that change formats or start/stop getting returned)

Reply:
Got it. That makes sense.